### PR TITLE
Make compile and run again

### PR DESCRIPTION
**Pull request by peppy**
Autoplay and playfield alignment needs fixing still.

Also I broke references a bit, I'll fix those. PRing for comment and hopefully some collaboration on fixing the playfield (not sure what's going on at a glance).

### CODE REVIEW

File: osu.Game.Rulesets.Pippidon/UI/PippidonPlayfield.cs [hunk 1]
```diff
@@ -117,16 +118,17 @@ private set
 
             public Texture Texture
             {
-                set
-                {
-                    ((Sprite)Child).Texture = value;
-                }
+                set { ((Sprite)Child).Texture = value; }
```
Comment (jorolf) [inline osu.Game.Rulesets.Pippidon/UI/PippidonPlayfield.cs:118]: `set => ...`?

Comment (peppy) [reply osu.Game.Rulesets.Pippidon/UI/PippidonPlayfield.cs:118]: feel free to change as you see fit. if you're going to start complaining about code styling, there's a *lot* more that needs fixing across this whole codebase :P

Comment (jorolf) [reply osu.Game.Rulesets.Pippidon/UI/PippidonPlayfield.cs:118]: you're probably right :P

Comment (jorolf) [reply osu.Game.Rulesets.Pippidon/UI/PippidonPlayfield.cs:118]: you're probably right :P

**Pull request merged**